<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxProvision">
       <property name="title">
        <string>Provision</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rPaste">
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rRice">
          <property name="text">
           <string>Rice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rFlour">
          <property name="text">
           <string>Flour</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxFruits">
       <property name="title">
        <string>Fruits</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rApple">
          <property name="text">
           <string>Apple</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBanana">
          <property name="text">
           <string>Banana</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rLeman">
          <property name="text">
           <string>Leman</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxKg">
       <property name="title">
        <string>Kg</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="rOne">
          <property name="text">
           <string>One Kg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rTwo">
          <property name="text">
           <string>Two Kg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rThree">
          <property name="text">
           <string>Three Kg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCoronoSecure">
       <property name="title">
        <string>Corona Secure</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="rGlove">
          <property name="text">
           <string>Glove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rMask">
          <property name="text">
           <string>Mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rCologne">
          <property name="text">
           <string>Cologne</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
